File: temp/progs/Степенко 12/program4.py
import math

n = int(input("Введите число членов ряда: "))

res = 0
for i in range(1, n+1):
    try:
        res += 3 ** math.log(i) / (i ** 2 - 1)
    except ZeroDivisionError as e:
        print("На итерации {} произошло исключение: {}".format(i, e))

print("Результат: ", res)

File: temp/progs/Флягин 9/program4.lua
io.write("Введите число членов ряда: ")
local n = io.read("*n")

local e = math.exp(1)
local res = 0
for i = 2, n+1 do -- при значении i = 1, переменная res принимает значение inf
  res = res + 1 / (i * math.log(i, e) ^ 2)
end

print(string.format("Результат: %f", res))
